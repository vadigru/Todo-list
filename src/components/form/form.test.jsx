import React from "react";
import renderer from "react-test-renderer";

import Form from "./form";

it(`Should render Form component`, () => {
  const tree = renderer
    .create(
        <Form
          onSubmitButtonClick={() => {}}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
