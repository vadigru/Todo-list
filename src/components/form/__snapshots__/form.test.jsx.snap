// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Form component 1`] = `
<form
  className="input"
  onSubmit={[Function]}
>
  <input
    autoComplete="off"
    className="input__text"
    id="input-task"
    onChange={[Function]}
    placeholder="input task"
    type="text"
    value=""
  />
  <label
    htmlFor="input-task"
  />
  <button
    className="input__submit"
    type="submit"
  >
    <img
      alt="add task"
      src="img/icons8-plus.svg"
    />
  </button>
</form>
`;
